shared/common-adapters/toast.d.ts

```diff
@@ -4,10 +4,10 @@ import {StylesCrossPlatform} from '../styles'
 
 export type Props = {
   children: React.ReactNode
-  className?: string | null
+  className?: string
   containerStyle?: StylesCrossPlatform
   visible: boolean
-  attachTo?: (() => any | null) | null
+  attachTo?: () => any | null
```

Comment:
Is this redundant? is `any | null` different from `any`?

Reply:
it is